dpe/src/dpe_instance.rs

```diff
@@ -32,12 +32,13 @@ pub struct DpeInstance<'a, C: Crypto, P: Platform> {
     // DpeInstance doesn't actually need to hold an instance. The PhantomData
     // is just to make the Crypto and Platform traits work.
     phantom_platform: PhantomData<P>,
+    phantom_crypto: PhantomData<C>,
 }
 
 impl<C: Crypto, P: Platform> DpeInstance<'_, C, P> {
     const MAX_NEW_HANDLE_ATTEMPTS: usize = 8;
 
-    pub(crate) fn new_context_handles() -> [Context<C>; MAX_HANDLES] {
+    pub(crate) fn new_context_handles() -> [Context; MAX_HANDLES] {
         core::array::from_fn(|_| Context::new())
     }
```

Comment:
nit: we could change this back to the `const` implementation we used to have

```Rust
const CONTEXT_INITIALIZER: Context = Context::new();
        let mut dpe = DpeInstance {
            contexts: [CONTEXT_INITIALIZER; MAX_HANDLES],
```

Reply:
Done.